src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -230,7 +231,7 @@ def load_multi_cluster_config(self):
 
     @Endpoint('PUT')
     @UpdatePermission
-    def set_config(self, config: object):
+    def set_config(self, config: Dict[str, Any]):
```

Comment:
do we need to do this here? it looks unrelated. I don't think this will break anything but since we don't have an e2e for multi-cluster yet, its better to skip this to a separate commit or a PR of its own..

Reply:
@nizamial09 , I was getting mypy error related to this in make-check so had to make this change. Am not sure if a separate PR just for this change makes sense.